src/CSSList/CSSList.php

```diff
@@ -296,6 +296,22 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     */
+    public function insertBefore($oItem, $oSibling): void
+    {
+        $iIndex = in_array($oSibling, $this->aContents);
+        if ($iIndex === false) {
+            $this->append($oItem);
+        } else {
+            $this->replace($oSibling, array($oItem, $oSibling));
```

Comment:
Let's use the short array notation:

```suggestion
            $this->replace($oSibling, [$oItem, $oSibling]);
```

Reply:
Yes.